packages/grafana-ui/src/components/Alert/Alert.story.tsx

```diff
@@ -21,172 +20,61 @@ const meta: ComponentMeta<typeof Alert> = {
     knobs: {
       disable: true,
     },
-    controls: {
-      exclude: ['onRemove'],
-    },
+    controls: {},
   },
   argTypes: {
     severity: { control: { type: 'select', options: severities } },
+    onRemove: {
+      description: "To show a button or a close icon, select 'function'. To get rid of them, select 'undefined'",
+      control: { type: 'select', options: [action('Remove button clicked'), undefined] },
+    },
```

Comment:
i'm not sure we should do this either tbh... feels like we should just make a new story `Inline Banner with onRemove` 🤷

Reply:
I've changed the code, so the `Inline Banner` shows three examples:
1. Without the close icon or button.
2. Just the close icon.
3. Just the button.

And the controls for this component are 'title', 'severity', and 'button' text. The 'button' text changes will only be shown in the corresponding example.

Regarding the `Toast` component, one example and 'title' and 'severity' as controls.